scripts/check-latest-tag.cjs

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const packageJSON = require('../package.json');
+const packageLockJSON = require('../package-lock.json');
+
+let tagVersion = process.env.GITHUB_REF_NAME || '';
+console.log('Checking tag:', tagVersion);
+
+if (!tagVersion.startsWith('v')) {
```

Comment:
seems like we haven't been prefixing `v`?

https://github.com/radarlabs/react-native-radar/releases

Reply:
Ahhh, removed from the beta check but not from the latest check. Good catch.

https://github.com/radarlabs/react-native-radar/pull/238/commits/517e2f48e29f6fc7a1596b83736dac4022b34e21